privaterelay/templates/includes/banners.html

```diff
@@ -25,12 +25,12 @@
     <div class="dismiss-wrapper">
       <button class="notification-dismiss js-dismiss"><img src="{% static 'images/x-close.svg' %}"></button>
     </div>
-    <div class="notification-banner-bg">
-      <div class="notification-banner-content">
-        <div class="notification-banner-copy">
-          <p class="notification-banner-header">{% ftlmsg 'banner-label-data-notification-header' %}</p>
-          <p class="notification-banner-sub">{% ftlmsg 'banner-label-data-notification-body' %}</p>
-          <a class="notification-banner-link" href="#" rel="noopener noreferrer">{% ftlmsg 'banner-label-data-notification-cta' %}</a>
+    <div class="data-notification-banner-bg">
+      <div class="data-notification-banner-content">
+        <div class="data-notification-banner-copy">
+          <p class="data-notification-banner-header">{% ftlmsg 'banner-label-data-notification-header' %}</p>
+          <p class="data-notification-banner-sub">{% ftlmsg 'banner-label-data-notification-body' %}</p>
+          <a class="data-notification-banner-link" href="{{ settings.FXA_SUBSCRIPTIONS_URL }}/products/{{ settings.PREMIUM_PROD_ID }}?plan={{ settings.PREMIUM_PRICE_ID }}" target="_blank" rel="noopener noreferrer">{% ftlmsg 'banner-label-data-notification-cta' %}</a>
```

Comment:
This actually needs to go to `/accounts/settings/`. Could you please update?

Reply:
Done!